answers.js

```diff
@@ -124,17 +124,16 @@ const bondFilms = [
 
 // oddBonds;
 
-let grossBond = [];
-let grossBondTotal = [];
 
-bondFilms.map( (item )=>{
-grossBond.push(item.gross.replace('$', ''));
-})
+let grossBond = bondFilms.map((item, i ) => {
+  return parseInt(item.gross.replace('$', '').replace(/,/gi, ''));
+});
 
-bondFilms.map((profit)=>{
-	parseInt(profit, 10);
-	grossBondTotal += profit;
-})
 
-grossBondTotal;
+let grossTotal = grossBond.reduce((total, amount) => {
+  return total + amount;
+});
+
+console.log(grossTotal);
```

Comment:
I'm glad you asked this question. You had the right instincts from the beginning!